### ISSUE TITLE
Regression (PrimeNG LTS 9.1.9): scrollable p-table throws error 'this.scrollableViewChild.clearCache'

### ISSUE THREAD

**Issue by j0hnwhyte** [primefaces/primeng#9558]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
I'm having issues trying to reproduce it in plunkr after installing the LTS library. As it didn't happen before, I presume it should be trivial to spot the code that changed since the last non-lts version. It happened right after I purchased and installed it.

**Current behavior**
When I create a table element (scrollable, flex), it throws the error:
```
TypeError: this.scrollableViewChild.clearCache is not a function
```

 This doesn't happen if the table isn't scrollable.

* **Angular version:** 9.1.7

* **PrimeNG version:** LTS 9.1.9

* **Browser:** all

**Comment by cagataycivici** [comment]
Please see;

https://github.com/primefaces/primeng/wiki/Migration-Guide

**Comment by j0hnwhyte** [comment]
Hi and thanks for the quick response. Unfortunately I'm not calling `clearCache` anywhere in my code, rather it's being done in `table.ts:506`. All I'm doing is feeding data to the table in my component.

**Comment by yigitfindikli** [comment]
Hi @j0hnwhyte fixed for [v9.2.0-lts](https://www.npmjs.com/package/primeng-lts/v/9.2.0) and [v10.0.5](https://www.npmjs.com/package/primeng-lts/v/10.0.5).
